Sources/SessionTools/KeychainStorage/KeychainStorageContainer.swift

```diff
@@ -8,14 +8,36 @@
 import Foundation
 import KeychainAccess
 
+/// The lifecycle of the keychain item added
+public enum KeychainLifecycle: Equatable {
+    /// Default case - it's up to you and the OS rules to define when this session is deleted
+    case unmanaged
+    
+    /// The session will only live in the current installation - a delete and reinstall will make this item unreachable.
+    /// - identifier: A unique identifier for the installation. It should remain stable between installations.
```

Comment:
Similar to a comment somewhere above. I think we would want `identifier` to remain stable between app launches, but unique between app installations.